integration/wai-aria.spec.js

```diff
@@ -0,0 +1,297 @@
+import path from 'path';
+import { spawnSync } from 'child_process';
```

Comment:
`spawnSync` doesn't seem to be being used at this point, so just wondering if it's necessary to import it?

Reply:
Good catch!